src/Microsoft.Performance.SDK.Runtime.Tests/Extensibility/TestClasses/TestSourceDataCooker.cs

```diff
@@ -33,12 +33,42 @@ public TestSourceDataCooker(TestSourceDataCookerContext context)
 
         public T QueryOutput<T>(DataOutputPath identifier)
         {
-            throw new System.NotImplementedException();
+            if (Reflector == null)
```

Comment:
Should we have a separate text fixture for the reflection tests?

Reply:
Perhaps, but I don't want to update for this PR as this was already being used.